services/app-api/src/emailer/emails/__snapshots__/newPackageCMSEmail.test.ts.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders overall email as expected 1`] = `
+"Managed Care submission: <b>CMS-new-submission-snapshot</b> was received from <b>MN</b>.
+<br /><br />
+<b>Submission type</b>: Contract action and rate certification<br />
+<b>Contract effective dates</b>: 07/05/2022 to 07/05/2022<br />
+<b>Rate name</b>: CMS-new-submission-snapshot-RATE-20220705-20220705-CERTIFICATION-20220705<br />
+<b>Rating period<b>: 07/05/2022 to 07/05/2022<br />
+<b>Submission description</b>: A submitted submission<br />
+<br />
+<a href=\\"http://localhost/submissions/test-abc-125\\">View submission</a>
```

Comment:
🔍  Looks like there might be an escaping quotes issue going on with the view submission link. I will take a look at this